src/lib/messenger.js

```diff
@@ -34,7 +34,8 @@ export default function messenger(bot) {
       } else if (inputParser.isAskingForBoegivar(text)) {
         handleBoegivar(bot, message.chat, user.firstName);
       } else if (inputParser.isAskingForTourettes(text)) {
-        handleTourettes(bot, message.chat, text);
+        const commandText = text.substr(11, text.length);
```

Comment:
I think it would be nicer to create a helper function to strip the `text` from the command itself, in that way it's possible to abstract out unnecessary details at a first glance. If the reader later wants to know _how_ the command is stripped out he/she can only go to the helper function for those details.